src/main/java/Task.java

```diff
@@ -146,15 +141,11 @@ public DeadlineTask(String[] inputArr) throws Exception {
                     this.time = LocalDateTime.parse(this.description.substring(lastBy + 4),
                             DateTimeFormatter.ofPattern("MMM d yyyy hh:mma"));
                 } catch (DateTimeParseException e2) {
-                    System.out.println("Error: unable to decipher date & time input.");
-                    e.printStackTrace();
+                    throw new Exception("Error: unable to decipher date & time input.");
```

Comment:
Might want to consider also telling the user what the expected DateTime format is, so they can key it in properly the next time round?

Reply:
When the program runs it displays the list of commands. Date format is under that.